packages/core/src/base-axis-chart.ts

```diff
@@ -260,6 +264,18 @@ export class BaseAxisChart extends BaseChart {
 			.text(this.options.scales.x.title);
 	}
 
+	repositionYAxisTitle() {
+		const yAxisRef = this.svg.select("g.y.axis");
+		const tickHeight = this.getLargestTickHeight(yAxisRef.selectAll(".tick"));
+
+		const yAxisTitleRef = this.svg.select("g.y.axis text.y.axis-label");
+		yAxisTitleRef.attr("class", "y axis-label")
+		.attr("text-align", "center")
+		.attr("transform", `rotate(-90) translate(${ - (yAxisRef.node().getBBox().height / 2)},
```

Comment:
This should all be on the same line

Reply:
All of that will exceed that max line length

Reply:
which means that the logic should be broken down into different variables...